src/theme/notFound.scss

```diff
@@ -0,0 +1,15 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
```

Comment:
I don't think we should be adding a new font just for this page. Let's just use the existing styles.

Reply:
as requested I removed the fonts.